### PR TITLE
Turn off 'unicorn/no-nested-ternary'

### LINKED ISSUE #110: 'unicorn/no-nested-ternary' conflict with Prettier
[eslint-plugin-unicorn](https://github.com/sindresorhus/eslint-plugin-unicorn)'s [no-nested-ternary](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/no-nested-ternary.md) rule is conflict with Prettier. It's added in [v11](https://github.com/sindresorhus/eslint-plugin-unicorn/releases/tag/v11.0.0).

### PR DESCRIPTION
**Pull request by yangmingshan**
Fix #110

### CODE REVIEW

File: unicorn.js [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 module.exports = {
   rules: {
-    "unicorn/number-literal-case": "off"
+    "unicorn/number-literal-case": "off",
```
Comment (lydell) [inline unicorn.js]: The rules in these files are sort alphabetically :)

```suggestion
    "unicorn/no-nested-ternary": "off",
```

File: unicorn.js [hunk 2]
```diff
@@ -2,6 +2,7 @@
...
+    "unicorn/no-nested-ternary": "off"
```
Comment (lydell) [inline unicorn.js]:
```suggestion
    "unicorn/number-literal-case": "off"
```

**Comment by yangmingshan** [comment]
@lydell Updated. I just accidentally used a work account, so I force pushed. Sorry about that.

**Comment by lydell** [comment]
Thanks! It turned out easier to do it myself in the end, so that’s what I ended up doing.

**Pull request closed without merging**